lib/phoenix/router.ex

```diff
@@ -210,15 +210,15 @@ defmodule Phoenix.Router do
 
   Scopes allow us to scope on any path or even on the helper name:
 
-      scope "/v1", MyAppWeb, host: "api." do
+      scope "/api/v1", MyAppWeb, as: :api_v1 do
```

Comment:
Updated to revert the example change from [this PR](https://github.com/phoenixframework/phoenix/commit/16f2ef38057c12f68636bfe9350340a165785b68#diff-c474801b0de930e7c3fd5808258094655afe8b008149ff17bee4caaf0d85a154R182), which doesn't match the explanation